### DISCUSSION TITLE
Welcome to pdf-bullets Discussions!

### DISCUSSION THREAD

**Discussion by ckhordiasma** [AF-VCD/pdf-bullets#39] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by NYakel** [comment]
Is there a specific format for importing files to the "abbreviations" tab? Tried adding new abbreviations a couple different ways and wasn't successful. Any help would be much appreciated!

**Reply by Bordep** [reply]
Three columns.
1. Should just be the word TRUE in caps
2. The full word
3. The abbreviation

Let me know if that doesn't work for you or if you'd like me to send you an example.

**Comment by jahatfi** [comment]
@ckhordiasma I see there's some open Pull Requests that haven't been touched since you stepped away.  You mentioned in your README message about no longer working for the Air Force that you don't intend to increase/decrease support for this page.  Do you have any plans either approve the PR or delegate permission to do so?  Or at this point if we still like this tool is it best to fork it from here?  Thanks!

**Comment by ckhordiasma** [comment]
Thanks for the ping, I'm taking a look. 

I am going to try to shoot for PR reviews on a monthly basis, if you would like for things to move faster than that then by all means fork and I won't be offended!

**Comment by em48126** [comment]
Hi there! First and foremost, your tool has helped me make so many packages over these last few year, so big thanks to you. You're a frickin' hero. Here is what I have going on. The Air Force, as you're well aware, uses Adobe for all of it's forms and locks them from modification. This makes it difficult to use proper spacing and fonts in other programs that are actually shareable and simultaneously editable. My current goal is to make a word/excel/googledoc/something I can toss in teams/sharepoint that is real-time editable by the leads in my shops so we don't have to play the "here's my version of the 1206" email game. I've tried doing a Table set up in word with .25" margins, but the Times New Roman 12 pt. font doesn't line up exactly the same way the TNR in Adobe does due to what I assume is spacing (especially lower case"f".) Excel was a disaster from the get-go when it came to font matching. Do you have any ideas on how I would be able to make something like this work?
